src/libcore/cmp.rs

```diff
@@ -95,6 +96,76 @@ use self::Ordering::*;
 /// assert!(b1 != b3);
 /// ```
 ///
+/// ## How can I implement it to compare with another type?
```

Comment:
This reads as awkward to me; can we get one last tweak?

    /// ## How can I compare two different types?